packages/config/src/api/site_info.ts

```diff
@@ -52,42 +69,52 @@ export const getSiteInfo = async function ({
   return { siteInfo, accounts, addons, integrations }
 }
 
-const getSite = async function (api, siteId, siteFeatureFlagPrefix = null) {
+const getSite = async function (api: NetlifyAPI, siteId: string, siteFeatureFlagPrefix: string | null = null) {
   if (siteId === undefined) {
     return {}
   }
 
   try {
-    const site = await api.getSite({ siteId, feature_flags: siteFeatureFlagPrefix })
+    const site = await (api as any).getSite({ siteId, feature_flags: siteFeatureFlagPrefix })
     return { ...site, id: siteId }
   } catch (error) {
     throwUserError(`Failed retrieving site data for site ${siteId}: ${error.message}. ${ERROR_CALL_TO_ACTION}`)
   }
 }
 
-const getAccounts = async function (api) {
+const getAccounts = async function (api: NetlifyAPI) {
   try {
-    const accounts = await api.listAccountsForUser()
+    const accounts = await (api as any).listAccountsForUser()
     return Array.isArray(accounts) ? accounts : []
   } catch (error) {
     throwUserError(`Failed retrieving user account: ${error.message}. ${ERROR_CALL_TO_ACTION}`)
   }
 }
 
-const getAddons = async function (api, siteId) {
+const getAddons = async function (api: NetlifyAPI, siteId: string) {
   if (siteId === undefined) {
     return []
   }
 
   try {
-    const addons = await api.listServiceInstancesForSite({ siteId })
+    const addons = await (api as any).listServiceInstancesForSite({ siteId })
     return Array.isArray(addons) ? addons : []
   } catch (error) {
     throwUserError(`Failed retrieving addons for site ${siteId}: ${error.message}. ${ERROR_CALL_TO_ACTION}`)
   }
 }
 
-const getIntegrations = async function ({ api, ownerType, ownerId, testOpts }) {
+type GetIntegrationsOpts = {
+  ownerType: 'site' | 'team'
+  ownerId: string | undefined
+  testOpts: TestOptions
+}
+
+const getIntegrations = async function ({
+  ownerType,
+  ownerId,
```

Comment:
mind renaming it to `siteId` as it's the site id?

Reply:
yeah, need to ditch the `ownerType` requirement anyway as we're scoping it to sites - shouldn't have left it so wide